tests/publish-instance.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { test } from "./fixtures/lxd-test";
+import { deleteAllImages } from "./helpers/images";
+import {
+  createInstance,
+  deleteInstance,
+  randomInstanceName,
+  visitInstance,
+} from "./helpers/instances";
+
+test("Publish an image from an instance", async ({ page }) => {
+  const instance = randomInstanceName();
+  await createInstance(page, instance);
+
+  await page.goto("/ui/");
+  await visitInstance(page, instance);
+  await page.getByRole("button", { name: "Publish" }).click();
+  await page
+    .getByRole("dialog", { name: "Create image from instance" })
+    .getByRole("button", { name: "Create image" })
+    .click();
+
+  await page.waitForSelector(`text=Image published from instance ${instance}.`);
+
+  await deleteAllImages(page);
+  await deleteInstance(page, instance);
+  await page.close();
```

Comment:
last comment for the day, you don't need to call `page.close()` here since we are using standard `test` functionality here.